client/ayon_maya/plugins/load/load_redshift_proxy.py

```diff
@@ -148,3 +153,40 @@ def create_rs_proxy(self, name, path):
             cmds.setAttr("{}.useFrameExtension".format(rs_mesh), 1)
 
         return nodes, group_node
+
+    def _set_rs_proxy_file_type(self, proxy: str, path: str):
+        """Set Redshift Proxy file type attribute based on input file."""
+
+        extension = os.path.splitext(path)[1].lower()
```

Comment:
Would it be better to do this instead?
```suggestion
        extension = os.path.splitext(path)[-1].lower()
```

Reply:
Splitext always returns a two-tuple, so should be fine either way.